src/screens/KeyManagement/OCR2KeysCard.tsx

```diff
@@ -13,32 +13,42 @@ import { ConfirmationDialog } from 'src/components/Dialogs/ConfirmationDialog'
 import { LoadingRow } from 'src/components/TableRow/LoadingRow'
 import { NoContentRow } from 'src/components/TableRow/NoContentRow'
 import { OCR2KeyBundleRow } from './OCR2KeyBundleRow'
+import Button from '@material-ui/core/Button'
+import { OCR2KeysCreate } from 'screens/KeyManagement/OCR2KeysCreate'
 
 export interface Props {
   loading: boolean
   data?: FetchOcr2KeyBundles
   errorMsg?: string
   onDelete: (id: string) => Promise<any>
+  onCreate: (chainType: string) => Promise<any>
 }
 
-/**
- * This card follows the form and structure of OCRKeysCard but
- * does NOT yet offer a 'create' button as there are architecture
- * decisions TBD because OCR2 keys require association with a
- * chain family (e.g. EVM, Starknet, Solana) - but that list of
- * chains is becoming a more and more fluid/dynamic collection
- * and we need to consider how to offer this information from
- * the core to a client (in this case the operator UI)
- */
-export const OCR2KeysCard: React.FC<Props> = ({ data, loading, onDelete }) => {
+export const OCR2KeysCard: React.FC<Props> = ({
+  data,
+  loading,
+  onDelete,
+  onCreate,
+}) => {
   const [confirmDeleteID, setConfirmDeleteID] = React.useState<string | null>(
     null,
   )
+  const [toggleCreateOCR2KeyDialog, setToggleCreateOCR2KeyDialog] =
```

Comment:
```suggestion
  const [showOCR2CreationDialog, setShowOCR2CreationDialog] =
```
This isnt technically a toggle, since you can choose which state for it to be in, rather than only being able to invoke it to toggle the state to be the inverse of the previous state

Reply:
Changed in [`6bb1ac3` (#38)](https://github.com/smartcontractkit/operator-ui/pull/38/commits/6bb1ac37ef5ef0b55deb0414fff2035857755311)